relayer/src/chain/cosmos.rs

```diff
@@ -1133,6 +1134,11 @@ impl Chain for CosmosSdkChain {
         let connection_end = ConnectionEnd::decode_vec(&res.value)
             .map_err(|e| Kind::Query("proven connection".into()).context(e))?;
 
+        debug!(
+            "Gotten proven connection respons for connection {} at height {}: {:#?}",
```

Comment:
We usually try to make all log output start with a a lowercase.

```suggestion
            "got proven connection response for connection {} at height {}: {:#?}",
```

Reply:
Done